client/src/main/java/com/alibaba/nacos/client/naming/net/NamingProxy.java

```diff
@@ -327,7 +329,11 @@ public JSONObject sendBeat(BeatInfo beatInfo, boolean lightBeatEnabled) throws N
         Map<String, String> params = new HashMap<String, String>(8);
         String body = StringUtils.EMPTY;
         if (!lightBeatEnabled) {
-            body = "beat=" + JSON.toJSONString(beatInfo);
+            try {
+                body = "beat=" + URLEncoder.encode(JSON.toJSONString(beatInfo), "UTF-8");
+            } catch (UnsupportedEncodingException e) {
+                throw new NacosException(NacosException.SERVER_ERROR, "encod beatInfo error", e);
```

Comment:
"encode" not "encod"

Reply:
sorry, it's a spelling mistake. It's been fixed.